src/views/WaterTrackingView.jsx

```diff
@@ -4,36 +4,38 @@ import Title from "antd/es/typography/Title";
 const { Header, Content } = Layout;
 const { useMessage } = message;
 
-const WATER_GOAL = 100;
+const WATER_GOAL = 8;
 
 const createWaterChangeMessage = (messageInstance, value) => {
   if (value < 0) {
-    messageInstance.success(`Removed ${value} glass of water`);
+    messageInstance.error(`Removed ${value} glass of water`);
     return;
   }
 
-  messageInstance.error(`Added ${value} glass of water`);
+  messageInstance.success(`Added ${value} glass of water`);
 };
 
 const getProgressColor = (currentIntake, goal) => {
-  return "#50C878";
+  
 
   if (currentIntake < (2 / 100) * goal) {
```

Comment:
The percentage is wrong. Here are the correct rules:
should be red if the water is below 25% of the goal
should be orage below 50% of the goal
should be yellow below 75% of the goal
should be green above 75%